test/lib/marshal/guide/processor.test.ts

```diff
@@ -68,14 +112,6 @@ describe("lib/marshal/guide/processor", () => {
               },
             },
           ],
-          __readonly: {
-            key: "success-banner",
-            valid: true,
-            active: true,
-            environment: "development",
-            created_at: "2023-09-18T18:32:18.398053Z",
-            updated_at: "2023-10-02T19:24:48.714630Z",
-          },
```

Comment:
Q: Based on what we discussed in [this slack thread](https://knocklabs.slack.com/archives/C090NTVQAH4/p1749741942120599), I thought we were going to remove the updated_at and sha fields vs removing the __readonly field completely. Was there a change in thinking?